core-rust-bridge/src/main/java/com/radixdlt/statecomputer/commit/NextEpoch.java

```diff
@@ -64,14 +64,12 @@
 
 package com.radixdlt.statecomputer.commit;
 
-import com.google.common.collect.ImmutableMap;
-import com.radixdlt.rev2.ComponentAddress;
 import com.radixdlt.sbor.codec.CodecMap;
 import com.radixdlt.sbor.codec.StructCodec;
 import com.radixdlt.utils.UInt64;
+import java.util.Set;
 
-public record NextEpoch(
-    ImmutableMap<ComponentAddress, ActiveValidatorInfo> validators, UInt64 epoch) {
```

Comment:
_(note: it may seem weird why this had to change, so: Right now, we pass `NextEpoch` both ways, and it just happens that on Java->Rust way, the `ComponentAddress` can be `null` [not suitable for a map key]. This is demonstrated e.g. in `test_valid_rev2_transaction_passes()`. Maybe it's just the test that should be adjusted?)_